maven-release-manager/src/test/resources/pomfinder/pom1.xml

```diff
@@ -193,7 +193,7 @@
       <plugin>
         <groupId>org.codehaus.plexus</groupId>
         <artifactId>plexus-maven-plugin</artifactId>
-        <version>1.3.8</version>
+        <version>@plexusMavenPluginVersion@</version>
```

Comment:
Note: This likely needs to be rewritten in the future with Sisu and annotations.